src/platform/w3dengine/client/w3dparticlesys.cpp

```diff
@@ -13,13 +13,12 @@
  *            LICENSE
  */
 #include "w3dparticlesys.h"
+#include "particlesysmanager.h"
 #ifdef GAME_DLL
 #include "hooker.h"
 #endif
 
 void Do_Particles(RenderInfoClass &rinfo)
 {
-#ifdef GAME_DLL
-    Call_Function<void, RenderInfoClass &>(0x00778D20, rinfo);
-#endif
+    g_theParticleSystemManager->Do_Particles(rinfo);

```

Comment:
There was a nullptr check on this. Unsure if its needed.

Reply:
fixed.